docs/test_caggs.md

```diff
@@ -114,3 +115,14 @@ ORDER BY week;
  2020-07-20 00:00:00+00 |   168
  2020-07-27 00:00:00+00 |    59
 ```
+
+```SQL
+SELECT average(rollup(twa)) as time_weighted_average 
+FROM weekly_aggs;
+```
+
+```output
+ time_weighted_average
+-----------------------
+     67.0032679739
```

Comment:
Did you validate this result?

Reply:
I did but I also ended up deleting this test from caggs since it did not actually test the newly stabilized features.